<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>BkgndCfgBase</class>
  <widget class="QWidget" name="Form1" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>419</width>
        <height>187</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item rowspan="1" row="5" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item>
            <widget class="LinkLabel" name="lblLink" />
          </item>
          <item>
            <spacer name="Spacer4" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item row="6" column="1" >
        <spacer name="Spacer3" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="4" column="0" >
        <widget class="QLabel" name="TextLabel1_3" >
          <property name="text" stdset="0" >
            <string>Group margin:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="TextLabel1_2" >
          <property name="text" stdset="0" >
            <string>Contact margin:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="1" column="0" colspan="2" >
        <widget class="EditFile" name="edtPicture" />
      </item>
      <item rowspan="1" row="0" column="0" colspan="2" >
        <widget class="QLabel" name="TextLabel1" >
          <property name="text" stdset="0" >
            <string>Main window background:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignLeft</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="1" >
        <layout class="QHBoxLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item>
            <widget class="QSpinBox" name="spnContact" >
              <property name="maximum" stdset="0" >
                <number>20</number>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer9" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item row="4" column="1" >
        <layout class="QHBoxLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item>
            <widget class="QSpinBox" name="spnGroup" >
              <property name="maximum" stdset="0" >
                <number>20</number>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer4_2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item row="2" column="0" >
        <widget class="QLabel" name="TextLabel2" >
          <property name="text" stdset="0" >
            <string>Place:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="1" >
        <widget class="QComboBox" name="cmbPosition" />
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>QWidget</class>
      <extends>QWidget</extends>
      <header location="global" >QWidget</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
      <properties>
        <property type="String" >text</property>
      </properties>
    </customwidget>
    <customwidget>
      <class>LinkLabel</class>
      <extends>QWidget</extends>
      <header location="local" >simgui/linklabel.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>1</hordata>
        <verdata>1</verdata>
      </sizepolicy>
      <pixmap>image1</pixmap>
      <properties>
        <property type="String" >text</property>
      </properties>
    </customwidget>
    <customwidget>
      <class>EditFile</class>
      <extends>QWidget</extends>
      <header location="local" >simgui/editfile.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image1</pixmap>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
    <image name="image1" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1ddec44f503c0ae2a154410f53d0ed20e2bf6bdb656dd6861dd23d9a66591b0587fd1654235ebded6f0edcd53e419d87ae7b1f4f9b8f906d0bfe012317426a70b07bdc2f3ec77f8ed6b89559061a0343d06a124cc105596482585094bc0ae599b04646c9018926491b2205e140c485cace25755c175d0a967b622ff900b8cc9c7d29af594ea722d589167f813aa852ba07d94b9dce296e883fe7bb163f23896753</data>
    </image>
  </images>
</ui>
